src/app/contexts/UserContext/cookies/index.js

```diff
@@ -25,5 +25,13 @@ export const setPreferredVariantCookie = (service, variant) => {
 
   const variantCookieName = `ckps_${getVariantCookieName(service)}`;
   const COOKIE_EXPIRY = 7;
-  setCookie(variantCookieName, variant, COOKIE_EXPIRY);
+
+  // Setting variant cookie to have sameSite=Strict because the purpose of this
+  // cookie is to only be used in a first-party context
+  setCookie({
+    name: variantCookieName,
+    value: variant,
+    expires: COOKIE_EXPIRY,
+    sameSite: 'Strict',
```

Comment:
Variant cookie is only used on `.com` so we keep this to first party.

Reply:
Does using 'Strict' instead of 'Lax' mean that cookies aren't sent with the initial navigation to the site in a context where the user has followed a link from another site? Could this mean that variants aren't set correctly on the initial page view if a user comes from an external site?

Reply:
My understanding of the subject comes from https://web.dev/samesite-cookies-explained/

Reply:
Good catch Tom, on further thought I think we do want `Lax` here.